data/fixtures/recorded/scopes/boundedParagraph/chuckShortBlock.yml

```diff
@@ -0,0 +1,30 @@
+languageId: plaintext
+command:
+  version: 7
+  spokenForm: chuck short block
+  action:
+    name: remove
+    target:
+      type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: boundedParagraph}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    {
+
+        foo
+        bar
+    }
+  selections:
+    - anchor: {line: 2, character: 4}
+      active: {line: 2, character: 4}
+  marks: {}
+finalState:
+  documentContents: |-
+    {
+    }
```

Comment:
as mentioned elsewhere, I'd be tempted to close these `{}` but I don't feel strongly